### DISCUSSION TITLE
❤复❤制❤就❤看❤邦❤布❤可❤爱❤录❤像❤

### DISCUSSION THREAD

**Discussion by Laiyucha** [share121/inter-knot#3018] [其他]
![image](https://github.com/user-attachments/assets/d32cd471-969e-48ec-b3ad-f2f6e058a370)

bilibili.com/video/BV1im42137GG/?spm_id_from=333.999.0.0&vd_source=01293affa8c70b7acb32a6eec6f9ee77

**Comment by lllliufeng** [comment]
看满离

**Comment by ybbgh** [comment]
<a href="https://www.bilibili.com/video/BV17e8ceeEGq/?spm_id_from=333.788.recommend_more_video.5&vd_source=9f1c67940ef7fee251eca4e9f4089507" target="_blank">❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤</a>

**Comment by Guessme404** [comment]
再给我来100丁尼的！

**Comment by HauUhang** [comment]
[❤复❤制❤就❤看❤邦❤布❤可❤爱❤录❤像❤](bilibili.com/video/BV1im42137GG/?spm_id_from=333.999.0.0&vd_source=01293affa8c70b7acb32a6eec6f9ee77)<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
